### DISCUSSION TITLE
AC conversion consequences for other software

### DISCUSSION THREAD

**Discussion by WohnW** [OpenHantek/OpenHantek6022#325] [Q&A] [answered]
Hello, I would like to know how AC coupling modification (as described here https://github.com/OpenHantek/OpenHantek6022/blob/main/docs/HANTEK6022_AC_Modification.pdf ) impact the scope functionality with other software (eg Hantek stock software or Android Hscope software). Will device be still usable with other software?

**Comment by Ho-Ro** [comment, marked as answer]
As I do not use Windows I cannot test the Hantek stock SW. I also do not use Hscope. It uses the sigrok settings for A/DC and works under Pulseview if you enable the AC coupling with a [small config change](https://github.com/OpenHantek/OpenHantek6022/discussions/249).
